home.html

```diff
@@ -1,322 +1,420 @@
                     </div>
-                    <div class="txtcontainer">
-                      <p>CSS</p>
+                    <div class="grid-item">
+                      <div class="imgcontainer2">
+                        <img
+                          src="assets/html5logo.png"
+                          alt=""
+                          width="35"
+                          height="35"
+                        />
+                      </div>
+                      <div class="txtcontainer">
+                        <p>HTML</p>
+                      </div>
+                    </div>
+                    <div class="grid-item">
+                      <div class="imgcontainer3">
+                        <img
+                          src="assets/csslogo.png"
+                          alt=""
+                          width="35"
+                          height="35"
+                        />
+                      </div>
+                      <div class="txtcontainer">
+                        <p>CSS</p>
+                      </div>
                     </div>
                   </div>
-                </div>
-              </li>
-              <li class="listitem">
-                <div class="txtcontainer">
-                  <h3>Fameworks</h3>
-                </div>
-                <div class="imgcontainer">
-                  <img src="assets/collapse.svg" alt="" width="20" height="20" />
-                </div>
-              </li>
-              <li class="listitem">
-                <div class="txtcontainer">
-                  <h3>Skills</h3>
-                </div>
-                <div class="imgcontainer">
-                  <img src="assets/collapse.svg" alt="" width="20" height="20" />
-                </div>
-              </li>
-            </ul>
+                </li>
+                <li class="listitem">
+                  <div class="txtcontainer">
+                    <h3>Fameworks</h3>
+                  </div>
+                  <div class="imgcontainer">
+                    <img
+                      src="assets/collapse.svg"
+                      alt=""
+                      width="20"
+                      height="20"
+                    />
+                  </div>
+                </li>
+                <li class="listitem">
+                  <div class="txtcontainer">
+                    <h3>Skills</h3>
+                  </div>
+                  <div class="imgcontainer">
+                    <img
+                      src="assets/collapse.svg"
+                      alt=""
+                      width="20"
+                      height="20"
+                    />
+                  </div>
+                </li>
+              </ul>
+            </div>
           </div>
-        </div>
-      </section>
-
-    </main>
-  </div>
-</body>
-
-</html>
\ No newline at end of file
+        </section>
+        <section class="contact-container">
+          <form class="contact-form">
```

Comment:
- [ ] The Form should use the `action` and `method` attributes to integrate with [formspree service](https://formspree.io/html).